src/changelog-fetcher.js

```diff
@@ -70,17 +70,17 @@ class ChangelogFetcher {
 
     const prs = await this.client.paginate(options);
 
-    return chain(prs)
+    return _.chain(prs)
```

Comment:
```suggestion
    return _(prs)
```

On other uses of `_.chain()` as well.